nd4j/nd4j-backends/nd4j-tests/src/test/java/org/nd4j/autodiff/samediff/listeners/ProfilingListenerTest.java

```diff
@@ -45,8 +45,8 @@ public void testProfilingListenerSimple() throws Exception {
         SDVariable label = sd.placeHolder("label", DataType.FLOAT, 1, 2);
         SDVariable w = sd.var("w", Nd4j.rand(DataType.FLOAT, 3, 2));
         SDVariable b = sd.var("b", Nd4j.rand(DataType.FLOAT, 1, 2));
-        SDVariable sm = sd.nn.softmax("predictions", in.mmul("matmul", w).add("addbias", b));
-        SDVariable loss = sd.loss.logLoss("loss", label, sm);
+        SDVariable sm = sd.nn.softmax("predictions", in.mmul("matmul", w).add("addbias", b),0);
+        SDVariable loss = sd.loss.logLoss("loss", label, sm, null, 0.0);
```

Comment:
As per previous comemnts re: softmax and logLoss

Reply:
Fixed.